test/test_namedtuple_return_api.py

```diff
@@ -56,6 +56,7 @@ def test_namedtuple_return(self):
                names=('values', 'indices'), hasout=True),
             op(operators=['kthvalue'], input=(1, 0),
                names=('values', 'indices'), hasout=True),
+            op(operators=['msort'], input=(), names=('values', 'indices'), hasout=True),
```

Comment:
msort will be removed from this list, too, once it returns only the tensor

Reply:
Removed!